analysis/topeft_run2/run_analysis.py

```diff
@@ -273,7 +276,7 @@ def LoadJsonToSampleName(jsonFile, prefix):
             # forever until a larger worker connects.
             'resource_monitor': True,
             'resources_mode': 'auto',
-
+            'filepath': '/tmp',
```

Comment:
We _might_ need to remove this. I suppose most/all systems have `/tmp` so maybe it doesn't hurt. What do you think @Andrew42?